contracts/NFTAuction.sol

```diff
@@ -34,10 +34,10 @@ contract NFTAuction {
     /*
      * Default values that are used if not specified by the NFT seller.
      */
-    uint256 public defaultBidIncreasePercentage;
-    uint256 public defaultAuctionBidPeriod;
-    uint256 public minimumSettableIncreasePercentage;
-    uint256 public maximumMinPricePercentage;
+    uint32 public defaultBidIncreasePercentage;
+    uint32 public minimumSettableIncreasePercentage;
+    uint32 public maximumMinPricePercentage;
+    uint128 public defaultAuctionBidPeriod;
```

Comment:
@MJYoung114 Since there aren't in a struct, I think they aren't actually packed together, which means I think they all still use a 256 bit slot :D Which means they actually use more gas since they need to be type casted implicitly to uint256 xD 

Maybe you can double check this?